package.json

```diff
@@ -7,10 +7,10 @@
   "scripts": {
     "prepublish": "npm run build",
     "lint": "xo",
-    "dev": "npm run build && node $NODE_DEBUG_OPTION cli.js",
-    "build": "webpack",
-    "test": "ava && npm run lint",
-    "start": "npm run build && node cli.js"
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "test": "ava ./dist/test.js && npm run lint",
```

Comment:
Ah.. is the test transpiled? Ava already does this.

Reply:
Cool! Didn't know.